precompile/contracts/ibc/client_test.go

```diff
@@ -0,0 +1,550 @@
+package ibc
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ava-labs/subnet-evm/core/rawdb"
+	"github.com/ava-labs/subnet-evm/core/state"
+	"github.com/ava-labs/subnet-evm/core/vm"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/precompile/allowlist"
+	"github.com/cosmos/cosmos-sdk/codec"
+	cosmostypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/std"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/ethereum/go-ethereum/common"
+
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	commitmenttypes "github.com/cosmos/ibc-go/v7/modules/core/23-commitment/types"
+	"github.com/cosmos/ibc-go/v7/modules/core/exported"
+	ibctm "github.com/cosmos/ibc-go/v7/modules/light-clients/07-tendermint"
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+)
+
+func (suite *KeeperTestSuite) TestCreateClient() {
+	cases := []struct {
+		msg            string
+		clientState    exported.ClientState
+		consensusState exported.ConsensusState
+		expPass        bool
+		nextClientSeq  uint64
+	}{
+		{
+			"success: 07-tendermint client type supported",
```

Comment:
This is missing the "client type not supported" test case that's in the tendermint examples. Should that be here?

Reply:
fixed